twilio-audio-router/src/main/java/com/twilio/audio_manager/AudioDevice.java

```diff
@@ -0,0 +1,32 @@
+package com.twilio.audio_manager;
+
+import androidx.annotation.NonNull;
+
+public class AudioDevice {
+    public final @NonNull String name;
+    public final @NonNull Type type;
+
+    public AudioDevice(@NonNull Type type, @NonNull String name) {
```

Comment:
I think we can make this package protected since the audio device selector is responsible for discovering and enumerating devices.